patches/minecraft/net/minecraft/client/renderer/entity/LivingRenderer.java.patch

```diff
@@ -29,6 +29,17 @@
           f8 = MathHelper.func_219799_g(p_225623_3_, p_225623_1_.field_184618_aE, p_225623_1_.field_70721_aZ);
           f5 = p_225623_1_.field_184619_aG - p_225623_1_.field_70721_aZ * (1.0F - p_225623_3_);
           if (p_225623_1_.func_70631_g_()) {
+@@ -109,6 +112,9 @@
+       boolean flag2 = !flag1 && !p_225623_1_.func_98034_c(Minecraft.func_71410_x().field_71439_g);
+       this.field_77045_g.func_225597_a_(p_225623_1_, f5, f8, f7, f2, f6);
+       ResourceLocation resourcelocation = this.func_110775_a(p_225623_1_);
++      net.minecraftforge.client.event.RenderLivingEvent.Texture<T, M> event = new net.minecraftforge.client.event.RenderLivingEvent.Texture<>(p_225623_1_, this, p_225623_3_, p_225623_4_, resourcelocation);
++      net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(event);
++      resourcelocation = event.getTextureLocation();
```

Comment:
I don't know what the base for this patch file was, but this creates broken code. There is no `resourceLocation` variable in this method.

Reply:
There is one in line 35, though?

Reply:
Which doesn't exist in the actual code.
This patch changes `func_225623_a_`, which does not use `ResourceLocation` at all. This is what the file looks like after this patch is applied (achieved by checking out the PR "from" branch and running setup):
https://i.imgur.com/ZIQQArI.png

Reply:
That's very strange, I'll try cleaning my workspace